docs/source/intro/about.rst

```diff
@@ -4,17 +4,17 @@
 What is AiiDA
 *************
 
-Implement data-driven and fully automated workflows with AiiDA.
+AiiDA is an open-source Python infrastructure to automate, manage, persist, search and reproduce the complex workflows associated with modern computational science, and their associated data.
```

Comment:
If changed in `index.rst` we should adopt this accordingly

Reply:
Done